1116 catalog texts and placeholders

minump opened the pull request:
## Description
Changed form field and placeholder wordings for contribute form.

**Fixes #1116**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

sandeep-ps commented on contributions/catalog/webapps/templates/contribute/contribute.html:
I'm updating the Google Doc. I'll let you know soon.

sandeep-ps replied:
Hi, @minump, I have updated the Google doc. Those are changes like adding a period at the end of each line of text for consistency and adding more placeholder text for missing items.

minump replied:
updated text fields.

sandeep-ps replied:
Thank you!

minump commented:
"Select the Contribution Type and Add Details". --- this was "Select the Contribution Type" before. Since the label is for "Add Talent" and "Add Capability" buttons, do we need to change the label?

sandeep-ps commented:
> "Select the Contribution Type and Add Details". --- this was "Select the Contribution Type" before. Since the label is for "Add Talent" and "Add Capability" buttons, do we need to change the label?
> The one lien description for "Add a Capability" label has been delete. Is the description for "Add a Talent" label is also deleted? As of now, in the google docs it is "Add a Talent (a User Interface Component that Provides a Specific Functionality)"

I thought that when a user clicks that button, it's not just like selecting an option; a user needs to add more details. It's not very important, I think.
I have updated the suggestions in the Google doc to fix the inconsistencies. Please see. Thanks for catching that.

minump commented:
> > "Select the Contribution Type and Add Details". --- this was "Select the Contribution Type" before. Since the label is for "Add Talent" and "Add Capability" buttons, do we need to change the label?
> > The one lien description for "Add a Capability" label has been delete. Is the description for "Add a Talent" label is also deleted? As of now, in the google docs it is "Add a Talent (a User Interface Component that Provides a Specific Functionality)"
> 
> I thought that when a user clicks that button, it's not just like selecting an option; a user needs to add more details. It's not very important, I think. I have updated the suggestions in the Google doc to fix the inconsistencies. Please see. Thanks for catching that.

Updated labels and placeholders. 
Ready for review.

sandeep-ps reviewed (approved):
Thank you. The current changes look good to me. Approving.

minump commented:
Merging to develop
